CHANGELOG

```diff
@@ -1,6 +1,9 @@
 qtile next_version:
-    !!! extention subpackage renamed to extension !!!
-    !!! extentions option renamed to extensions !!!
+    !!! `extention` subpackage renamed to `extension` !!!
+    !!! `widget_defaults` and `extentions` options merged into `extension_defaults` !!!
```

Comment:
...however the idea to use `extension_defaults` was based on the possibility to restructure the bar as an extension on day. More ideas, if we want to agree to call the extension+bar group as "plugins" or "add-ons", the variable could be called `plugin_defaults` or `addon_defaults`.